router/fastdp.go

```diff
@@ -209,25 +209,25 @@ func (lock *fastDatapathLock) relock() {
 	}
 }
 
-// Bridge bits
+// InjectorConsumer bits
 
-type fastDatapathBridge struct {
+type fastDatapathInjectorConsumer struct {
```

Comment:
Are you talking about `func (fastdp *FastDatapath) bridge(...)` ?

That is a member of another struct, although exposed by Go's anonymous struct rules.